src/main/java/com/github/jmchilton/blend4j/galaxy/HistoriesClient.java

```diff
@@ -37,6 +38,20 @@
   HistoryExport exportHistory(String historyId);
   
   /**
+   * Deletes the given History from Galaxy (this will not purge).
+   * @param historyId  The id of the History to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteHistoryResponse(String historyId);
```

Comment:
Can you add something like that in the comments for those methods, explaining why you might want to use one over the other?